developers_chamber/ecs_utils.py

```diff
@@ -193,25 +193,64 @@ def start_service(cluster, service, count, region):
     except ecs_client.exceptions.ClusterNotFoundException:
         raise ClickException("Cluster not found: '{}'".format(cluster))
     except ecs_client.exceptions.ServiceNotFoundException:
-        raise ClickException("Service not found: '{}'".format(cluster))
+        raise ClickException("Service not found: '{}'".format(service))
     except ClientError as ex:
         raise ClickException(ex)
 
 
-def start_services(cluster, count, region):
+def start_services(cluster, services, count, region):
+    ecs_client = _get_ecs_client(region)
+
+    if count is not None and count < 1:
+        raise ClickException("Count must be greater than zero or 'None' to be determined from autoscaling targets.")
+
+    for service in services:
+        count_per_service = count
+        # service = service.split('/')[1]
+        if count_per_service is None:
+            try:
+                count_per_service = get_min_capacity_for_service(cluster=cluster, service=service, region=region)
+            except ClickException as ex:
+                raise ClickException(
+                    'Set explicit count or set up autoscaling with minimum capacity.\n{}'.format(ex)
+                )
+
+        LOGGER.info('Starting service: {}'.format(service))
+        try:
+            ecs_client.update_service(
+                cluster=cluster,
+                service=service,
+                desiredCount=count_per_service,
+            )
+        except ecs_client.exceptions.ClusterNotFoundException:
+            raise ClickException("Cluster not found: '{}'".format(cluster))
+        except ecs_client.exceptions.ServiceNotFoundException:
+            raise ClickException("Service not found: '{}'".format(service))
+        except ClientError as ex:
+            raise ClickException(ex)
+
+
+def is_service_type(service, cluster, type, region, ecs_client=None):
+    ecs_client = ecs_client if ecs_client else _get_ecs_client(region)
+    return ecs_client.describe_services(services=[service], cluster=cluster)['services'][0]['schedulingStrategy'] == type
+
+
+def start_cluster_services(cluster, count, region):
     ecs_client = _get_ecs_client(region)
 
     services = get_services_arns(cluster=cluster, region=region)
+    services = [service for service in services if not is_service_type(
+        service=service, cluster=cluster, type='DAEMON', region=region, ecs_client=ecs_client)]
 
     if count is not None and count < 1:
```

Comment:
Tyhle dva řádky se tu minimálně dvakrát opakují, nestálo by za to je sjednotit do jedné (třeba private) fce?
Jinak platí to, co jsem psal výše.

Reply:
Navíc, jestli dobře vidím, by kontrola countu klidně mohla být na začátku celé fce - pak by se zbytečně neprováděl ten kód nad ní, pokud by měla být raisnuta (count je rovnou na vstupu) - tedy za předpokladu, že ho některé z těch fcí nad ním v zákulisí nemění.